### DISCUSSION TITLE
Speech Background Noise Suppression with Deep Learning Project 193

### DISCUSSION THREAD

**Discussion by roaantm21** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#110] [Collaborate]
I am interested in working and collaborating on this project. I would like extra resources to help me improve my progress.Thank you!

**Comment by robertogl** [comment]
We already have a dedicated discussion for this project:
https://github.com/mathworks/MATLAB-Simulink-Challenge-Project-Hub/discussions/24
